packages/cli-kit/src/private/node/ui/components/TextAnimation.tsx

```diff
@@ -1,65 +1,47 @@
-import {renderString} from '../../ui.js'
-import chalkAnimation from 'chalk-animation'
+/* eslint-disable id-length */
 import {Text} from 'ink'
-import React, {useEffect, useState} from 'react'
-
-type AnimationName = 'rainbow' | 'pulse' | 'glitch' | 'radar' | 'neon' | 'karaoke'
+import React, {useEffect, useRef, useState} from 'react'
+import gradient from 'gradient-string'
```

Comment:
Not blocking, but these imports, especially React, [might have a negative impact](https://github.com/Shopify/cli/pull/970) on the startup time of the CLI. I'm working on a tool that will help us see if this is a bottleneck, and if it is we'll have to move them to dynamic imports.